src/application/utils/transaction.ts

```diff
@@ -80,6 +80,20 @@ async function blindPset(psetBase64: string, utxos: UnblindedOutput[], outputAdd
   ).toBase64();
 }
 
+function isFullyBlinded(psetBase64: string, excludeAddresses: string[]) {
+  const excludeScripts = excludeAddresses.map((a) => addrLDK.toOutputScript(a));
+  const tx = psetToUnsignedTx(psetBase64);
+  for (const out of tx.outs) {
+    if (out.script.length > 0 && !excludeScripts.includes(out.script)) {
+      if (!out.rangeProof || !out.surjectionProof) {
```

Comment:
not sure this is right check, should be .length > 1 ?

Reply:
script for fee outputs is an [emtpy buffer](https://github.com/vulpemventures/liquidjs-lib/blob/master/test/integration/transaction.spec.ts#L204), length > 0 looks good to me.